cms/envs/common.py

```diff
@@ -168,6 +168,7 @@
     # Enable support for content libraries. Note that content libraries are
     # only supported in courses using split mongo.
     'ENABLE_CONTENT_LIBRARIES': True,
```

Comment:
How do LIBRARIES_ENABLED and ENABLE_CONTENT_LIBRARIES differ? Are 2 flags actually necessary?